src/app/genes/gene-overview/exp-validation/exp-validation.component.html

```diff
@@ -12,8 +12,8 @@ <h2 *ngIf="data">Experimental Validation of {{d.hgnc_symbol}}</h2>
             <h1 class="inner-section-divider"></h1>
         </div>
         <div class="section-thin no-padding-top overview-sections">
-            <h2>{{teamInfo[i].program}}: {{teamInfo[i].team_full}}</h2>
-            <p class="col-8 section-desc">{{teamInfo[i].description}}</p>
+            <h2>{{getTeamInfoField(d.team, "program")}} : {{getTeamInfoField(d.team, "team_full")}}</h2>
```

Comment:
Each of these getTeamInfoField calls iterates over all items in teamInfo, so I'm sure there's a more efficient way to do this using Angular patterns.  Maybe wrap with something like:
<div *ngFor="let t of teamInfo;">
<div *ngIf="t.name === d.team ">
<h2>{{t.program}} : {{t.team_full}}</h2>
...

Reply:
This would iterate over teamInfo items once (instead three times) for every item in data.

Reply:
That does seem like a much better approach, thanks for the suggestion!